app/views/admin/resource/_index.html.erb

```diff
@@ -10,7 +10,10 @@
   <% collection.each do |resource| %>
     <tr>
       <% collection_attributes.each do |attribute| %>
-        <td><%= resource.send(attribute) %></td>
+        <td>
+          <%= resource.send(attribute) unless attribute == :grove %>
+          <%= resource.send(attribute).name if attribute == :grove %>
```

Comment:
this seems odd to put in the template... maybe we think about presenters or a custom template... I don't know.